src/init/features/apphosting/index.ts

```diff
@@ -212,9 +213,32 @@ export async function onboardRollout(
   const buildId = await apphosting.getNextRolloutId(projectId, location, backendId, 1);
   const buildOp = await apphosting.createBuild(projectId, location, backendId, buildId, buildInput);
 
-  const rolloutOp = await apphosting.createRollout(projectId, location, backendId, buildId, {
+  const rolloutBody = {
     build: `projects/${projectId}/locations/${location}/backends/${backendId}/builds/${buildId}`,
-  });
+  };
+
+  let tries = 0;
+  let done = false;
+  while (!done) {
+    tries++;
+    try {
+      await apphosting.createRollout(projectId, location, backendId, buildId, rolloutBody, true);
```

Comment:
can we include a arg comment /* validateOnly= */ true

Reply:
Blurg. I hate those. Named the value though, which helps.